### PR TITLE
refact(charts): add pod security policy for cstor charts

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

### CODE REVIEW

File: deploy/helm/charts/templates/psp.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{- if .Values.rbac.pspEnabled }}
```
Comment (akhilerm) [inline deploy/helm/charts/templates/psp.yaml:1]: Where are we using this PSP?

File: deploy/helm/charts/templates/csi-node.yaml [hunk 2]
```diff
@@ -55,8 +55,6 @@ spec:
         - name: {{ .Values.cstorCSIPlugin.name }}
           securityContext:
             privileged: true
-            capabilities:
-              add: ["CAP_MKNOD", "CAP_SYS_ADMIN", "SYS_ADMIN"]
```
Comment (akhilerm) [inline deploy/helm/charts/templates/csi-node.yaml:59]: :+1:

File: deploy/helm/charts/templates/psp.yaml (hunk 1 above)
Comment (prateekpandey14) [reply deploy/helm/charts/templates/psp.yaml:1]: missed to add rbac files

**Review by akhilerm — APPROVED** [review]
LGTM

**Pull request merged**